<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UyeIslemleri</class>
 <widget class="QDialog" name="UyeIslemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>182</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Üye No:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tumUyelerTablosu">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>301</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ÜYE İŞLEMLERİ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>609</x>
     <y>449</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userName">
   <property name="geometry">
    <rect>
     <x>602</x>
     <y>236</y>
     <width>191</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Üye Soyad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Üye Ad:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userSurname">
   <property name="geometry">
    <rect>
     <x>602</x>
     <y>300</y>
     <width>191</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newRecord">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Yeni Kayıt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>602</x>
     <y>172</y>
     <width>191</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>719</x>
     <y>390</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Güncelle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tüm Üyeler:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
